go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/gardener/etcd-druid
 go 1.22
 
 require (
-	github.com/gardener/etcd-backup-restore v0.29.0
+	github.com/gardener/etcd-backup-restore v0.26.0
```

Comment:
Why are you downgrading the `etcd-backup-restore` version from `v0.29.0` to `v0.26.0`

Reply:
This was a rebasing issue, fixed it now.